daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -495,13 +495,13 @@ enum BuiltinFunction {
   APPEND_TEXT = 24;
 
   ERROR = 25;
-  ANY_EXCEPTION_MESSAGE = 146;      // *Available in versions >= 1.dev*
-  MAKE_GENERAL_ERROR = 147;         // *Available in versions >= 1.dev*
-  GENERAL_ERROR_MESSAGE = 148;      // *Available in versions >= 1.dev*
-  MAKE_ARITHMETIC_ERROR = 149;      // *Available in versions >= 1.dev*
-  ARITHMETIC_ERROR_MESSAGE = 150;   // *Available in versions >= 1.dev*
-  MAKE_CONTRACT_ERROR = 151;        // *Available in versions >= 1.dev*
-  CONTRACT_ERROR_MESSAGE = 152;     // *Available in versions >= 1.dev*
+  ANY_EXCEPTION_MESSAGE = 147;      // *Available in versions >= 1.dev*
```

Comment:
Why change the existing numbers? I know it’s only 1.dev so this is fine but still doesn’t seem all that useful.

Reply:
I just find that cleaner